apps/wakunode2/config.nim

```diff
@@ -53,7 +53,6 @@ type
 
     nodekey* {.
       desc: "P2P node private key as 64 char hex string.",
-      defaultValue: defaultPrivateKey()
       name: "nodekey" }: PrivateKey
```

Comment:
If we are not providing a default value, then let's use a `Option[PrivateKey]`:
```suggestion
      name: "nodekey" }: Option[PrivateKey]
```

Reply:
Addressed in 1b9566c8